src/pages/Profile.tsx

```diff
@@ -51,11 +51,14 @@ const Profile = () => {
       icon: { iconName: IconName.Gear },
       href: AppRoute.Settings,
     },
-    // {
-    //   key: 'tutorials',
-    //   label: stringGetter({ key: STRING_KEYS.TUTORIALS }),
-    //   icon: { iconName: IconName.Comment },
-    // },
+    {
```

Comment:
<img width="440" alt="Screen Shot 2023-11-10 at 7 51 24 PM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/41455532-ce37-42e7-a45d-fa50a8a0dbd9">

conditionally render when Account is connected